core/testcontainers/core/generic.py

```diff
@@ -70,8 +70,12 @@ def _create_connection_url(
     def start(self) -> "DbContainer":
         self._configure()
         super().start()
+        self._transfer_seed()
         self._connect()
         return self
 
     def _configure(self) -> None:
         raise NotImplementedError
+
+    def _transfer_seed(self) -> None:
```

Comment:
To be discussed if this is a universal-enough feature to warrant this in DbContainer, as opposed to supporting it on a per-db basis.